spec/controllers/newflow/login_signup_controller_spec.rb

```diff
@@ -813,6 +802,21 @@ module Newflow
           expect(response).to redirect_to(root_url)
         end
       end
+
+      context 'when the stored url includes `redirect_uri` param' do
+        before do
+          allow_any_instance_of(described_class).to receive(:stored_url).and_return(subject)
+        end
+
+        subject do
+          "#{host}?redirect_uri=#{target_url}"
```

Comment:
this looks more like a let to me? (the subject is the actual object under test, the controller in this case)

let(:redirect_uri) { ... }

Reply:
Oh, true. I am still not always sure which is the subject under test.